package.json

```diff
@@ -58,8 +58,7 @@
     "eslint-config-okonet": "^5.0.1",
     "eslint-plugin-node": "^5.1.1",
     "husky": "^0.14.3",
-    "jest": "^21.1.0",
-    "jest-cli": "^21.1.0",
+    "jest": "^22.0.4",
     "jsonlint": "^1.6.2",
     "prettier": "1.9.2",
     "remove-lockfiles": "^1.1.1"
```

Comment:
`remove-lockfiles` needs to be removed if we agree to commit lockfile in this repo

Reply:
I figured we could deal with that in a separate PR..

Reply:
Don't think so since every time we try to commit the file it's being removed.

Reply:
What I meant was, a separate PR could remove the `remove-lockfiles` and add the lock file.

Reply:
@okonet @sudo-suhas @SimenB I'll make a PR to remove `remove-lockfiles` and add the lock file after shalat. then we can continue working on this PR